### PR TITLE
Ac 519 human readable output for tx

### PR DESCRIPTION
**Pull request by Mayhallaj**
Fixed inconsistencies in cli output to allow scripting.  Created scripts for testing cli command in accordance with the test matrix.

### CODE REVIEW

File: scripts/cli_create_id.sh [hunk 1]
```diff
@@ -2,19 +2,18 @@
...
-if [ -z $1 ]; then
-	echo "You must enter an IPAddress:Port for a server to generate an account"
-	exit 0
-fi
-
 # issue the account generate command to the specified server
 
-ID=`$cli account generate -s "http://$1/v1"`
+if [ -z $1 ]; then
+	acc="$($cli account generate -j 2>&1 > /dev/null | /usr/bin/jq .name | /usr/bin/sed 's/\"//g')"
```
Comment (bunfield) [inline scripts/cli_create_id.sh:10]: on ubuntu 18.04 sed is located in /bin/sed

File: scripts/test_case_2.6.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# test case 2.3
+#
+# create an adi from another adi
+# server IP:Port needed unless defaulting to localhost
+#
+# set cli command and see if it exists
+#
+export cli=../cmd/cli/cli
```
Comment (bunfield) [inline scripts/test_case_2.6.sh:10]: probably should ensure the cli is built, maybe?  

go run ../cmd/cli/main.go might be an option, but that will be very slow if doing loops.

**Review by bunfield — CHANGES_REQUESTED** [review]
This PR should be cancelled in favor of AC-237-scripts-for-version-0-2

**Comment by Mayhallaj** [comment]
Doesn't the script prompt the user to build the cli if it isn't there?  Who gives a crap about Ubuntu... I guess the script could check and see if which sed returns a valid file name.
    On Tuesday, November 16, 2021, 03:24:40 PM CST, bunfield ***@***.***> wrote:  
 
 


@bunfield requested changes on this pull request.

This PR should be cancelled in favor of AC-237-scripts-for-version-0-2

In scripts/cli_create_id.sh:
>  # issue the account generate command to the specified server
 
-ID=`$cli account generate -s "http://$1/v1"`
+if [ -z $1 ]; then
+	acc="$($cli account generate -j 2>&1 > /dev/null | /usr/bin/jq .name | /usr/bin/sed 's/\"//g')"

on ubuntu 18.04 sed is located in /bin/sed

In scripts/test_case_2.6.sh:
> @@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# test case 2.3
+#
+# create an adi from another adi
+# server IP:Port needed unless defaulting to localhost
+#
+# set cli command and see if it exists
+#
+export cli=../cmd/cli/cli

probably should ensure the cli is built, maybe?

go run ../cmd/cli/main.go might be an option, but that will be very slow if doing loops.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Mayhallaj** [comment]
Wrong branch.

**Pull request closed without merging**